git/cmd.py

```diff
@@ -609,6 +609,12 @@ def execute(self, command,
         # end handle
 
         try:
+            if sys.platform == 'win32':
```

Comment:
Generally for these kinds of assignment, I believe it's easier to read by starting out like this:
```python
creationflags = None
if sys.platform == 'win32':
	creationflags = CREATE_NO_WINDOW
```